app/code/Magento/GoogleGtag/Helper/GtagConfiguration.php

```diff
@@ -0,0 +1,153 @@
+
+    /**#@-*/
+
+    /**#@+
+     * Google AdWords conversion src
+     */
+    public const GTAG_GLOBAL_SITE_TAG_SRC = 'https://www.googletagmanager.com/gtag/js?id=';
+
+    /**#@+
+     * Google AdWords config data
+     */
+    public const XML_PATH_ADWORD_ACTIVE = 'google/gtag/adwords/active';
+
+    public const XML_PATH_CONVERSION_ID = 'google/gtag/adwords/conversion_id';
+
+    public const XML_PATH_CONVERSION_LABEL = 'google/gtag/adwords/conversion_label';
+
+    /**
+     * Whether GA is ready to use
+     *
+     * @param null|string|bool|int|Store $store
+     * @return bool
+     */
+    public function isGoogleAnalyticsAvailable($store = null): bool
+    {
+        return $this->scopeConfig->isSetFlag(
+            self::XML_PATH_ACTIVE,
+            ScopeInterface::SCOPE_STORE,
+            $store
+        ) && $this->getMeasurementId();
+    }
+
+    /**
+     * Get Account Id, depending on property type Tracking Id (UA) or Measurement Id (GA4)
+     *
+     * @return string
+     */
+    public function getMeasurementId(): string
+    {
+        return (string)$this->scopeConfig->getValue(
+            self::XML_PATH_MEASUREMENT_ID,
+            ScopeInterface::SCOPE_STORE
+        );
+    }
+
+    /**
+     * Is Google AdWords active
+     *
+     * @return bool
+     */
+    public function isGoogleAdwordsActive(): bool
+    {
+        return $this->scopeConfig->isSetFlag(
+            self::XML_PATH_ADWORD_ACTIVE,
+            ScopeInterface::SCOPE_STORE
+        ) &&
+            $this->getConversionId() &&
+            $this->getConversionLabel();
+    }
+
+    /**
+     * Is Google AdWords congifurable
+     *
+     * @return bool
+     */
+    public function isGoogleAdwordsConfigurable(): bool
+    {
+        return $this->scopeConfig->isSetFlag(
+            self::XML_PATH_ADWORD_ACTIVE,
+            ScopeInterface::SCOPE_STORE
+        ) && $this->getConversionId();
+    }
+
+    /**
+     * Get conversion js src
+     *
+     * @return string
+     */
+    public function getConversionGtagGlobalSiteTagSrc(): string
+    {
+        $siteSrc = self::GTAG_GLOBAL_SITE_TAG_SRC;
+        $cId = $this->getConversionId();
+        return $siteSrc . $cId;
+    }
+
+    /**
+     * Get Google AdWords conversion id
+     *
+     * @return string
+     */
+    public function getConversionId(): string
+    {
+        return (string)$this->scopeConfig->getValue(
+            self::XML_PATH_CONVERSION_ID,
+            ScopeInterface::SCOPE_STORE
+        );
+    }
+
+    /**
+     * Get Google AdWords conversion label
+     *
+     * @return string
+     */
+    public function getConversionLabel(): string
+    {
+        return $this->scopeConfig->getValue(
+            self::XML_PATH_CONVERSION_LABEL,
+            ScopeInterface::SCOPE_STORE
+        );
+    }
+
+    /**
+     * Format Data
+     *
+     * @param float $numberString
+     * @return Float
+     */
+    public function formatToDec($numberString)
```

Comment:
Can this just be used in place? Do we really need a public method (say `@api`) for this operation located in the arbitrary helper?

Reply:
Removed it and directly use this function in Block